packages/frontend/src/mantineTheme.ts

```diff
@@ -113,6 +113,11 @@ export const getMantineThemeOverride = (overrides?: {
                                 ),
                             },
                     },
+                    viewport: {
+                        '& > div': {
+                            display: 'block !important', // Only way to override the display value (from `table`) of the Viewport's child element
+                        },
+                    },
```

Comment:
This change would remove horizontal scroll in all ScrollAreas.
We can apply this behaviour via a classname:

```
            <ScrollArea variant="primary" offsetScrollbars scrollbarSize={8} className={'only-vertical'}>
```

```
                    viewport: {
                        '.only-vertical & > div': {
                            display: 'block !important', // Only way to override the display value (from `table`) of the Viewport's child element
                        },
                    },
```